### PR TITLE
Fixes softgloves

### LINKED ISSUE #3109: Softgloves work at a range
testing confirmed, soft gloves work even if your target is 2+ tiles away.

nerf plz

### PR DESCRIPTION
**Pull request by ghost**
fixes #3109 

:cl: Thefastfoodguy
fix: Soft gloves no longer work when far away
/:cl:

### CODE REVIEW

File: hippiestation/code/modules/clothing/gloves/softgloves.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 /obj/item/clothing/gloves/color/white/soft/Touch(mob/living/carbon/human/target,proximity)
 	var/mob/M = loc
-	if(ishuman(target))
+	if(ishuman(target) && Adjacent(target))
 		if(M.a_intent == INTENT_HELP && target.a_intent != INTENT_HELP)
 			target.a_intent = INTENT_HELP
 			target.hud_used.action_intent.icon_state = "[target.a_intent]" //Else we get your intent being one thing and your hud another.
```
Comment (ghost) [inline hippiestation/code/modules/clothing/gloves/softgloves.dm]: could we just use a_intent_change(INTENT_HELP) here instead?

**Review by ghost — DISMISSED** [review]
if this could be changed, it'd save snowflake code

File: hippiestation/code/modules/clothing/gloves/softgloves.dm [hunk 2]
```diff
@@ -4,10 +4,9 @@
 
 /obj/item/clothing/gloves/color/white/soft/Touch(mob/living/carbon/human/target,proximity)
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/clothing/gloves/softgloves.dm]: Try changing "proximity" to 1 and removing the &&adjacent. Touch() is called during attack_hand and should have its own prox check built in (which I totally failed to use)

File: hippiestation/code/modules/clothing/gloves/softgloves.dm [hunk 3]
```diff
@@ -4,10 +4,9 @@
...
 	var/mob/M = loc
-	if(ishuman(target))
+	if(ishuman(target) && Adjacent(target))
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/clothing/gloves/softgloves.dm]: Be sure to test this locally by possesing a mob, changing its intent to anything but help, giving it a soft glove hug with another mob and checking its intent selector icon properly changed.

File: hippiestation/code/modules/clothing/gloves/softgloves.dm (hunk 1 above)
Comment (KayeArray) [reply hippiestation/code/modules/clothing/gloves/softgloves.dm]: depends on whether touch is called by TK or not

Comment (KayeArray) [reply hippiestation/code/modules/clothing/gloves/softgloves.dm]: wait wrong line

**Review by j-mie — APPROVED** [review]

**Pull request merged**